sdks/java/harness/src/main/java/org/apache/beam/fn/harness/PrecombineGroupingTable.java

```diff
@@ -189,16 +184,10 @@ <K> WindowedGroupingTableKey(
         K key,
         Collection<? extends BoundedWindow> windows,
         Coder<K> keyCoder,
-        SizeEstimator<K> keySizer) {
+        SizeEstimator sizer) {
```

Comment:
Shouldn't this be a SizeEstimator<WindowedGroupingTableKey>? Or is there a reason size estimators are all untyped now?

Reply:
Caches::weigh doesn't care about type. It relies on Jamm which uses java's instrumentation API to measure the size of the object in memory.